app/views/forever_style_guide/sections/visual_elements/typography/_sizes.erb

```diff
@@ -0,0 +1,28 @@
+<h3 class="style-guide-partial-title">
+  Sizes
+</h3>
+<div class="style-guide-partial-rendered">
+  <table class="table">
+    <tbody>
+    <tr>
+      <td>
+        <label>.font-size-{size}</label>
+      </td>
+      <td>
+        <p class="font-size-100">Font Size 100</p>
+        <p class="font-size-200">Font Size 200</p>
+        <p class="font-size-300">Font Size 300</p>
+        <p class="font-size-400">Font Size 400</p>
+        <p class="font-size-500">Font Size 500</p>
+        <p class="font-size-600">Font Size 600</p>
+        <p class="font-size-700">Font Size 700</p>
+        <p class="font-size-800">Font Size 800</p>
+        <p class="font-size-lg">Font Size lg - same as 400</p>
```

Comment:
Ill double check that now, but I remember having issues last week with that. And then we had that conversation about class naming: 
![screen shot 2016-12-27 at 10 26 48 am](https://cloud.githubusercontent.com/assets/7773346/21502619/28221270-cc1f-11e6-9b2d-aa1a7f8c5721.png)